src/Microsoft.Azure.SignalR.Common/Auth/AzureADOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class AzureAdOptions
+    {
+        public bool Enabled = false;

```

Comment:
no public fields

Reply:
I think we can use a similar format as MADL client lib https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

Reply:
Good idea.